src/main/java/org/jabref/model/entry/BibEntry.java

```diff
@@ -489,7 +489,7 @@ private Optional<String> genericGetFieldOrAlias(Field field, BiFunction<BibEntry
             return date.map(Date::getNormalized);
         }
 
-        if (StandardField.YEAR == field || StandardField.MONTH == field || StandardField.DAY == field) {
+        if ((StandardField.YEAR == field) || (StandardField.MONTH == field) || (StandardField.DAY == field)) {
```

Comment:
Honestly, i find those additional brackets more annoying instead of helping.

Reply:
feel free to remove them.